modules/administration-guide/partials/proc_setting-up-the-gitlab-authorized-application.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: PROCEDURE
+:description: Setting up the GitLab authorized application
+:keywords: gitlab, gitlab-application, gitlab-authorized-application, authorized-application
+:navtitle: Setting up the GitLab authorized application
+// :page-aliases:
+
+[id="setting-up-the-gitlab-authorized-application_{context}"]
+= Setting up the GitLab authorized application
+
+Set up a GitLab authorized application using OAuth 2.0.
+
+.Prerequisites
+
+* You are logged in to GitLab.
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. Click your avatar and go to menu:Edit profile[Applications].
+
+. Enter *{prod-short}* as the *Name*.
+
+. Enter `pass:c,a,q[{prod-url}]/api/oauth/callback` as the *Redirect URI*.
+
+. Check the *Confidential* and *Expire access tokens* checkboxes.
+
+. Under *Scopes*, check the `api`, `write_repository`, and `openid` checkboxes.
+
+. Click *Save application*.
+
+. Copy the *GitLab Application ID* and encode it to Base64:
```

Comment:
Consider merging all remaining step into one single step:

* Convert *GitLab Application ID* and *GitLab Client Secret* to the Base64 format for later use in {prod-short}.

Reply:
Thanks for the suggestion.
I've merged two steps, yet differently:

![image](https://user-images.githubusercontent.com/78564304/163005442-a5214dea-f87e-43a9-84fd-7defd993540d.png)